modules/backend/classes/FormField.php

```diff
@@ -243,6 +247,24 @@ public function options($value = null)
         return $this;
     }
 
+    /**
+     * Returns the label(s) associated with a field value(s) for the field options
+     */
+    public function valueLabel(): string|array
```

Comment:
What is this used for?

Reply:
Good question! :rofl:

It's usefull if managing a Form widget that has fields with options (dropdown, etc) to extract the label associated with its value in the field's options.